airbyte-integrations/connectors/destination-databricks/src/test-integration/kotlin/io/airbyte/integrations/destination/databricks/typededupe/AbstractDatabricksTypingDedupingTest.kt

```diff
@@ -26,25 +26,25 @@ import org.junit.jupiter.api.BeforeAll
 import org.junit.jupiter.api.Timeout
 import org.mockito.Mockito
 
-class DatabricksTypingDedupingTest : BaseTypingDedupingTest() {
+abstract class AbstractDatabricksTypingDedupingTest(
+    private val jdbcDatabase: JdbcDatabase,
+    private val connectorConfig: DatabricksConnectorConfig,
+) : BaseTypingDedupingTest() {
     override val imageName: String
         get() = "airbyte/destination-databricks:dev"
 
     companion object {
-        private var jdbcDatabase: JdbcDatabase = Mockito.mock()
-        private var connectorConfig: DatabricksConnectorConfig = Mockito.mock()
```

Comment:
we need these variables to be instanced per connector config, so push them down into their respective class' companion object